yt_dlp/extractor/bigo.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    urlencode_postdata,
+    ExtractorError,
+)
+
+
+class BigoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bigo\.tv/(?:[a-z]{2,}/)?(?P<id>[^/]+)'
+    # Note: I would like to provide some real test, but Bigo is a live streaming
+    # site, and a test would require that the broadcaster is live at the moment.
+    # So, I don't have a good way to provide a real test here.
+    _TESTS = [{
+        'url': 'https://www.bigo.tv/th/Tarlerm1304',
+        'only_matching': True,
+    }, {
+        'url': 'https://bigo.tv/115976881',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        user_id = self._match_id(url)
+
+        INFO_URL = 'https://bigo.tv/studio/getInternalStudioInfo'
+        info_raw = self._download_json(
+            INFO_URL, user_id, data=urlencode_postdata({'siteId': user_id}))
```

Comment:
```suggestion
        info_raw = self._download_json(
            'https://bigo.tv/studio/getInternalStudioInfo', user_id, data=urlencode_postdata({'siteId': user_id}))
```
The variable seems unnecessary